### PR TITLE
revive atlas real

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
very first map i played on and first map ive been sus on, so i want to bring it back

**Main changes**
- replaced all access hangar thing with dorms and secret room
- proper grav gen + comms/camera room
- update everyones equipment
- hop office was awkward and in the way, and still is after the previous remaking, so moved it below bridge
- named all the power stuff and organized it better
- ikea atmos
- nav map gaming
- station records in places so det can do their job
- added faxes to each department
- arrivals dock
- bolting airlocks and add some firelocks to places
- add escape pod to north maints

**Changelog**
:cl:
- add: Atlas Station is back for lowpop rounds!

### CODE REVIEW

**Comment by Emisse** [comment]
my one atlas ss14 fan

**Comment by Emisse** [comment]
ill take a look at this today

**Comment by Emisse** [comment]
image
why are these out here
image
grass exposes wires and stuff now gotta move these around
image
dorm needs more decorations like carpet, posters, etc
image
move RND or Robotics to the left of RDs office so its not intermingled theres a lot of space to work with
image
this needs redoing 
image
ahoy
image
dawg just make this room bigger somehow
image
you cant walk out of this wihtout moving the locker
image
decalling here is kinda fucked up maybe do something like the boxes used in a lot of hop lines
image
lights on doors
image
probably just make proper telecomms on other side theres space
image
old idea for atlas i had was to give lots of supply for solar panels so engis can build arrays outside of it

TLDR
redo medical basically its outdated
cramped rooms need to be made less cramped
SCI needs some splitty uppy

good job

**Comment by deltanedas** [comment]
> why are these out here

i wanted them somewhere, kitchen was cramped and there was a bunch of empty space there. should i remove or expand kitchen?

> proper telecomms on other side theres space
which side?

**Comment by Emisse** [comment]
> 
> which side?

the south side i think you should jsut mirror the room

**Comment by Tunguso4ka** [comment]
no noticeboard? :(

**Comment by deltanedas** [comment]
i forgor :trollface:

**Comment by Emisse** [comment]
image

**Comment by deltanedas** [comment]
oh the disposals right i had no idea what that was showing :trollface:

**Comment by Emisse** [comment]
time to make the playerbase mad

**Comment by deltanedas** [comment]
:trollface:
